doc/sphinxdoc/models.rst

```diff
@@ -0,0 +1,653 @@
+----------
+
+Music classification by mood (2 classes):
+
+`party`, `non_party`
+
+Dataset: inhouse (MTG).
+
+Output: activations.
+
+Models:
+
+* ``mood_party-musicnn-msd``
+* ``mood_party-musicnn-mtt``
+* ``mood_party-vgg-msd``
+* ``mood_party-vgg-mtt``
+* ``mood_party-vggish-audioset``
+
+
+Mood Relaxed
+------------
+
+Music classification by mood (2 classes):
+
+`relaxed`, `non_relaxed`
+
+Dataset: inhouse (MTG).
+
+Output: activations.
+
+Models:
+
+* ``mood_relaxed-musicnn-msd``
+* ``mood_relaxed-musicnn-mtt``
+* ``mood_relaxed-vgg-msd``
+* ``mood_relaxed-vgg-mtt``
+* ``mood_relaxed-vggish-audioset``
+
+
+Mood Sad
+--------
+
+Music classification by mood (2 classes):
+
+`sad`, `non_sad`
+
+Dataset: inhouse (MTG).
+
+Output: activations.
+
+Models:
+
+* ``mood_sad-musicnn-msd``
+* ``mood_sad-musicnn-mtt``
+* ``mood_sad-vgg-msd``
+* ``mood_sad-vgg-mtt``
+* ``mood_sad-vggish-audioset``
+
+
+Moods MIREX
+-----------
+
+Music classification by mood (5 mood clusters):
+
+`1: passionate, rousing, confident, boisterous, rowdy`,
+`2: rollicking, cheerful, fun, sweet, amiable/good natured`,
+`3: literate, poignant, wistful, bittersweet, autumnal, brooding`,
+`4: humorous, silly, campy, quirky, whimsical, witty, wry`,
+`5: aggressive, fiery, tense/anxious, intense, volatile, visceral`
+
+Dataset: MIREX Audio Mood Classification Dataset
+
+Output: activations.
+
+Models:
+
+* ``moods_mirex-musicnn-msd``
+* ``moods_mirex-musicnn-mtt``
+* ``moods_mirex-vgg-msd``
+* ``moods_mirex-vgg-mtt``
+* ``moods_mirex-vggish-audioset``
+
+
+Tonal / Atonal
+--------------
+
+Music classification by tonality (classes):
+
+`tonal`, `atonal`
+
+Dataset: inhouse (MTG).
+
+Output: activations.
+
+Models:
+
+* ``tonal_atonal-musicnn-msd``
+* ``tonal_atonal-musicnn-mtt``
+* ``tonal_atonal-vgg-msd``
+* ``tonal_atonal-vgg-mtt``
+* ``tonal_atonal-vggish-audioset``
+
+
+Urban sound classification
+--------------------------
+
+Urban environment sound classification (10 classes):
+
+`air conditioner`, `car horn`, `children playing`, `dog bark`, `drilling`, `engine idling`, `gun shot`, `jackhammer`, `siren`, `street music`
+
+Dataset: UrbanSound8K.
+
+Output: activations.
+
+Models:
+
+* ``urbansound8k-musicnn-msd``
+
+
+Feature extractors
+^^^^^^^^^^^^^^^^^^
+
+Download: https://essentia.upf.edu/models/feature-extractors/
+
+
+OpenL3
+------
+
+Audio embeddings model trained in a self-supervised manner using audio-visual correspondence information.
+
+Dataset: AudioSet subsets of videos with environmental sounds and musical content.
+
+Output: embeddings.
+
+Models:
+
+* ``openl3-env-mel128-emb512``
+* ``openl3-env-mel128-emb6144``
+* ``openl3-env-mel256-emb512``
+* ``openl3-env-mel256-emb6144``
+* ``openl3-music-mel128-emb512``
+* ``openl3-music-mel128-emb6144``
+* ``openl3-music-mel256-emb512``
+* ``openl3-music-mel256-emb6144``
+
+
+AudioSet-VGGish
+---------------
+
+Audio embeddings model accompanying the AudioSet dataset, trained to predict tags from Youtube videos.
```

Comment:
For consistency with OpenL3: "Audio embeddings model accompanying the AudioSet dataset trained in a supervised manner using tag information for YouTube videos." ?